x-pack/platform/plugins/shared/alerting/server/alerts_service/alerts_service.ts

```diff
@@ -511,4 +512,164 @@ export class AlertsService implements IAlertsService {
       ...opts,
     });
   }
+
+  public async muteAlertInstance({
```

Comment:
Maybe a nit, but the size of the changes could be cut in half. `mute` and `unmuteAlertInstance` are very similar, and `mute/unmuteAllAlerts` too.

We could maybe reduce this to:

```
public async muteAlertInstance(params) {
    this._updateMuteState({value: true, ...params})
}

public async unmuteAlertInstance(params) {
    this._updateMuteState({value: false, ...params})
}
```